tests/phpunit/tests/block-templates/buildBlockTemplateResultFromFile.php

```diff
@@ -178,4 +192,78 @@ public function test_should_ignore_post_types_property_when_building_template_pa
 
 		$this->assertEmpty( $template->post_types );
 	}
+
+	/**
+	 * @ticket 60506
+	 */
+	public function test_should_inject_hooked_block_into_template_part() {
+		register_block_type(
+			'tests/my-block',
+			array(
+				'block_hooks' => array(
+					'core/paragraph' => 'after',
+				),
+			)
+		);
+
+		$template_part = _build_block_template_result_from_file(
+			array(
+				'slug'      => 'header',
+				'postTypes' => array( 'post' ),
+				'path'      => DIR_TESTDATA . '/templates/template.html',
+			),
+			'wp_template_part'
+		);
+		$this->assertStringEndsWith( '<!-- wp:tests/my-block /-->', $template_part->content );
+	}
+
+	/*
```

Comment:
Missing star. Same for some other unit tests

For ex. `/**`

Reply:
Good catch, thank you! I'll address in a follow-up.